src/wizard/dotnet/google-cloud-functions.md

```diff
@@ -0,0 +1,107 @@
+---
+name: Google Cloud Functions
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/google-cloud-functions/
+support_level: production
+type: framework
+---
+
+Install the **NuGet** package:
+
+Package Manager:
+
+```shell
+Install-Package Sentry.Google.Cloud.Functions -Version {{ packages.version('sentry.dotnet.google-cloud-function') }}
+```
+
+Or .NET Core CLI:
+
+```shell
+dotnet add package Sentry.Google.Cloud.Functions -v {{ packages.version('sentry.dotnet.google-cloud-function') }}
+```
+
+Add Sentry to `Function` class through the `FunctionsStartup`:
+
+
+```csharp
+// Add the following line:
+[assembly: FunctionsStartup(typeof(SentryStartup))]
+
+public class Function : IHttpFunction
+{
+    public Task HandleAsync(HttpContext context)
+    {
+        // Your function code here. 
+    }
+}
+```
+
+Additionally, you'll need to set up your `Sentry` settings on `appsettings.json`:
+
+```json
+{
+  "Sentry": {
+    "Dsn": "___PUBLIC_DSN___",
+    // Sends Cookies, User Id when one is logged on and user IP address to sentry. It's turned off by default.
+    "SendDefaultPii": true,
+    // Enable tracing.
+    "TracesSampleRate": 1.0,
+    // Opt-in for payload submission.
+    "MaxRequestBodySize": "Always"
+  }
+}
+```
+
+## Verify
+
+To verify your setup, you can capture a message with the SDK:
+
+```csharp
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Sentry;
+
+public Task HandleAsync(HttpContext context)
+{
+    SentrySdk.CaptureMessage("Hello Sentry");
+}
+```
+
+### Performance Monitoring
+
+You can measure the performance of your code by capturing transactions and spans.
+
+```csharp
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Sentry;
+
+public Task HandleAsync(HttpContext context)
+{
+    // Transaction can be started by providing, at minimum, the name and the operation
+    var transaction = SentrySdk.StartTransaction(
+    "test-transaction-name",
+    "test-transaction-operation"
+    );
+    SentrySdk.ConfigureScope(scope => scope.Transaction = transaction);
+
+    // Transactions can have child spans (and those spans can have child spans as well)
+    var span = transaction.StartChild("test-child-operation");
+
+    // ...
+    // (Perform the operation represented by the span/transaction)
+    // ...
+
+    span.Finish(); // Mark the span as finished
```

Comment:
In the event of an exception this won't be called an no transaction will be captured.

Ideally we'd make the integration start the transaction for the HTTP request and avoid this instruction altogether. We can then just show the StartChild here